src/themes/huraga/html/layout_default.html.twig

```diff
@@ -43,341 +33,237 @@
-            <a href="{{ settings.footer_link_3_page }}">{{ settings.footer_link_3_title }}</a>
-            {% else %}
-            <a href="{{ settings.footer_link_3_page | link }}">{{ settings.footer_link_3_title }}</a>
-
-            {%endif %}
-        </li>
-        {% endif %}
-        {% if settings.footer_link_4_enabled %}
-        <li>
-            {% if 'http://' in settings.footer_link_4_page or  'https://' in settings.footer_link_4_page%}
-            <a href="{{ settings.footer_link_4_page }}">{{ settings.footer_link_4_title }}</a>
-            {% else %}
-            <a href="{{ settings.footer_link_4_page | link }}">{{ settings.footer_link_4_title }}</a>
-            {% endif %}
-        </li>
-        {% endif %}
-        {% if settings.footer_link_5_enabled %}
-        <li>
-            {% if 'http://' in settings.footer_link_5_page or  'https://' in settings.footer_link_5_page%}
-            <a href="{{ settings.footer_link_5_page }}">{{ settings.footer_link_5_title }}</a>
-            {% else %}
-            <a href="{{ settings.footer_link_5_page | link }}">{{ settings.footer_link_5_title }}</a>
-            {% endif %}
-        </li>
-        {% endif %}
-
-        {% if guest.extension_is_on({"mod":'branding'}) %}
-        <li>
-            <a href="https://fossbilling.org" title="Billing Software" target="_blank">{{ 'Powered by the FOSSBilling Community'|trans }}</a>
-        </li>
-        {% endif %}
-    </ul>
+    </div>
+    <div class="toast-container position-fixed bottom-0 end-0 p-3" style="z-index: 1070;"></div>
     {% if settings.footer_to_top_enabled %}
-    <a href="#top" class="btn btn-primary btn-flat pull-right"><span class="awe-arrow-up"></span> {{ 'Top'|trans }}</a>
+        <a href="#top" class="position-fixed btn btn-primary bottom-0 end-0 m-3">
+            <span class="awe-arrow-up"></span> {{ 'Top'|trans }}</a>
     {% endif %}
-</footer>
-{% endif %}
-
-<div class="wait" style="display:none" onclick="$(this).hide();">
-    <div class="popup_block" style="position: fixed; top: 0; left: 0; width: 100%; height: 100%; background-color: black; opacity: 0.5; -webkit-opacity: 0.5; -moz-opacity: 0.5; filter :  alpha(opacity=50); z-index: 2000">
-        <img alt="loading gif" src="{{ 'img/loader.gif'|asset_url}}" style="position: absolute; display: block; margin-left: auto; margin-right: auto; position: relative; top: 50%; opacity: 1; filter: alpha(opacity=100); z-index: 1003">
+    <div class="wait" style="display:none" onclick="$(this).hide();">
+        <div class="spinner-border"
+             style="width: 4rem; height: 4rem; top: 50%; left: 50%; position: fixed; z-index: 999"></div>
     </div>
-</div>
-<noscript>NOTE: Many features on FOSSBilling require Javascript and cookies. You can enable both via your browser's preference settings.</noscript>
-
-{% endblock %}
+    <noscript>NOTE: Many features on FOSSBilling require Javascript and cookies. You can enable both via your browser's
```

Comment:
Maybe this should be translateable?

Reply:
I don't think we need to worry about it.
It's going to be quite uncommon for someone to have JS completely disabled and if they did items like the language selector wouldn't be functional anyways.